internal/resources/targetlocation/resource_schema.go

```diff
@@ -0,0 +1,203 @@
+package targetlocation
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
+)
+
+const (
+	ResourceName = "tanzu-mission-control_target_location"
+
+	// Root Keys
+	nameKey         = "name"
+	providerNameKey = "provider_name"
+	specKey         = "spec"
+
+	// Spec Directive Keys
+	bucketKey         = "bucket"
+	regionKey         = "region"
+	caCertKey         = "ca_cert"
+	assignedGroupsKey = "assigned_groups"
+	configKey         = "config"
+	credentialKey     = "credential"
+	targetProviderKey = "target_provider"
+
+	// Assigned Groups Directive Keys
+	clusterGroupsKey                 = "cluster_groups"
+	clusterKey                       = "cluster"
+	clustersManagementClusterNameKey = "management_cluster_name"
+	clustersProvisionerNameKey       = "provisioner_name"
+	clustersNameKey                  = "name"
+
+	// Config Directive Keys
+	awsConfigKey           = "aws"
+	awsS3ForcePathKey      = "s3_force_path_style"
+	awsS3BucketUrlKey      = "s3_bucket_url"
+	awsS3PublicUrlKey      = "s3_public_url"
+	azureConfigKey         = "azure"
+	azureResourceGroupKey  = "resource_group"
+	azureStorageAccountKey = "storage_account"
+	azureSubscriptionIdKey = "subscription_id"
+)
+
+var backupTargetLocationResourceSchema = map[string]*schema.Schema{
+	nameKey:         nameSchema,
+	providerNameKey: providerNameSchema,
+	specKey:         specSchema,
+	common.MetaKey:  common.Meta,
+}
+
+var nameSchema = &schema.Schema{
+	Type:     schema.TypeString,
+	Required: true,
+	ForceNew: true,
+}
+
+var providerNameSchema = &schema.Schema{
+	Type:     schema.TypeString,
+	Required: true,
+	ForceNew: true,
+}
+
+var specSchema = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "Spec of enable backup target location",
+	Required:    true,
+	MaxItems:    1,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			bucketKey: {
+				Type:        schema.TypeString,
+				Description: "Bucket as target location for backups",
+				ForceNew:    true,
+				Optional:    true,
+			},
+			regionKey: {
+				Type:        schema.TypeString,
+				Description: "CA cert",
```

Comment:
Change description for regionKey and targetProviderKey.

Reply:
Done